### PR TITLE
[zh] Sync changes from English site (5)

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -1578,7 +1607,7 @@ Service's `type`.
 当查找主机 `my-service.prod.svc.cluster.local` 时，集群 DNS 服务返回 `CNAME` 记录，
 其值为 `my.database.example.com`。
 访问 `my-service` 的方式与其他服务的方式相同，但主要区别在于重定向发生在 DNS 级别，而不是通过代理或转发。
-如果以后您决定将数据库移到群集中，则可以启动其 Pod，添加适当的选择器或端点以及更改服务的 `type`。
+如果以后你决定将数据库移到群集中，则可以启动其 Pod，添加适当的选择器或端点以及更改服务的 `type`。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/services-networking/service.md:1610]: 群集？

File: content/zh/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -1811,8 +1846,9 @@ through a load-balancer, though in those cases the client IP does get altered.
 iptables operations slow down dramatically in large scale cluster e.g 10,000 Services.
 IPVS is designed for load balancing and based on in-kernel hash tables. So you can achieve performance consistency in large number of Services from IPVS-based kube-proxy. Meanwhile, IPVS-based kube-proxy has more sophisticated load balancing algorithms (least conns, locality, weighted, persistence).
 -->
-在大规模集群（例如 10000 个服务）中，iptables 操作会显着降低速度。 IPVS 专为负载平衡而设计，并基于内核内哈希表。
-因此，您可以通过基于 IPVS 的 kube-proxy 在大量服务中实现性能一致性。
+在大规模集群（例如 10000 个服务）中，iptables 操作会显着降低速度。 IPVS
+专为负载平衡而设计，并基于内核内哈希表。
+因此，你可以通过基于 IPVS 的 kube-proxy 在大量服务中实现性能一致性。
 同时，基于 IPVS 的 kube-proxy 具有更复杂的负载平衡算法（最小连接，局部性，加权，持久性）。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/services-networking/service.md:1852]: 负载平衡

File: content/zh/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -1821,57 +1857,48 @@ IPVS is designed for load balancing and based on in-kernel hash tables. So you c
 Service is a top-level resource in the Kubernetes REST API. You can find more details
 about the API object at: [Service API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core).
 -->
-Service 是 Kubernetes REST API 中的顶级资源。您可以在以下位置找到有关A PI 对象的更多详细信息：
+Service 是 Kubernetes REST API 中的顶级资源。你可以在以下位置找到有关A PI 对象的更多详细信息：
```
Comment (tanjunchen) [inline content/zh/docs/concepts/services-networking/service.md:1860]: 有关A PI ？

File: content/zh/docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -1959,47 +1987,28 @@ SCTP is not supported on Windows based nodes.
...
 <!--
 * Read [Connecting Applications with Services](/docs/concepts/services-networking/connect-applications-service/)
 * Read about [Ingress](/docs/concepts/services-networking/ingress/)
 * Read about [Endpoint Slices](/docs/concepts/services-networking/endpoint-slices/)
 -->
-
 * 阅读[使用服务访问应用](/zh/docs/concepts/services-networking/connect-applications-service/)
 * 阅读了解 [Ingress](/zh/docs/concepts/services-networking/ingress/)
-* 阅读了解 [端点切片](/zh/docs/concepts/services-networking/endpoint-slices/)
+* 阅读了解[端点切片](/zh/docs/concepts/services-networking/endpoint-slices/)
```
Comment (tanjunchen) [inline content/zh/docs/concepts/services-networking/service.md:2013]: 端点切片？

File: content/zh/docs/concepts/services-networking/service.md [hunk 5]
```diff
@@ -79,13 +79,13 @@ Kubernetes {{< glossary_tooltip term_id="pod" text="Pod" >}} 是有生命周期
...
-(see [below](#services-without-selectors) for why you might want a Service
-_without_ a selector).
+by a {{< glossary_tooltip text="selector" term_id="selector" >}}.
+To learn about other ways to define Service endpoints,
+see [Services _without_ selectors](#services-without-selectors).
 -->
 Kubernetes Service 定义了这样一种抽象：逻辑上的一组 Pod，一种可以访问它们的策略 —— 通常称为微服务。
 这一组 Pod 能够被 Service 访问到，通常是通过 {{< glossary_tooltip text="选择算符" term_id="selector" >}}
-（查看[下面](#services-without-selectors)了解，为什么你可能需要没有 selector 的 Service）实现的。
+要了解如何定义服务末端，请参阅[不带选择算符的服务](#services-without-selectors)。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/services-networking/service.md:88]: 末端？

File: content/zh/docs/concepts/services-networking/service.md [hunk 6]
```diff
@@ -721,7 +707,14 @@ A cluster-aware DNS server, such as CoreDNS, watches the Kubernetes API for new
 Services and creates a set of DNS records for each one.  If DNS has been enabled
 throughout your cluster then all Pods should automatically be able to resolve
 Services by their DNS name.
+-->
+你可以（几乎总是应该）使用[附加组件](/zh/docs/concepts/cluster-administration/addons/)
+为 Kubernetes 集群设置 DNS 服务。
 
+支持群集的 DNS 服务器（例如 CoreDNS）监视 Kubernetes API 中的新服务，并为每个服务创建一组 DNS 记录。
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/concepts/services-networking/service.md:714]: Many occurrence in previous translation. Now all fixed.

File: content/zh/docs/concepts/services-networking/service.md (hunk 4 above)
Comment (tengqm) [reply content/zh/docs/concepts/services-networking/service.md:2013]: 英文写成了两个词： Endpoint Slices

File: content/zh/docs/concepts/services-networking/service.md [hunk 7]
```diff
@@ -1880,8 +1907,9 @@ of Kubernetes itself, that will forward connections prefixed with
 The load balancer will send an initial series of octets describing the
 incoming connection, similar to this example
 -->
+### PROXY 协议
 
-如果您的云提供商支持它，
+如果你的云提供商支持它，
 则可以在 LoadBalancer 模式下使用 Service 在 Kubernetes 本身之外配置负载均衡器，
 该负载均衡器将转发前缀为 [PROXY协议](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt)
```
Comment (howieyuen) [inline content/zh/docs/concepts/services-networking/service.md:1914]: PROXY 协议

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/concepts/services-networking/dual-stack.md [hunk 8]
```diff
@@ -176,13 +176,13 @@ On cloud providers which support IPv6 enabled external load balancers, setting t
...
+The use of publicly routable and non-publicly routable IPv6 address blocks is acceptable provided the underlying {{< glossary_tooltip text="CNI" term_id="cni" >}} provider is able to implement the transport. If you have a Pod that uses non-publicly routable IPv6 and want that Pod to reach off-cluster destinations (eg. the public Internet), you must set up IP masquerading for the egress traffic and any replies. The [ip-masq-agent](https://github.com/kubernetes-sigsr/ip-masq-agent) is dual-stack aware, so you can use ip-masq-agent for IP masquerading on dual-stack clusters.
 -->
 公共路由和非公共路由的 IPv6 地址块的使用是可以的。提供底层
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/concepts/services-networking/dual-stack.md:181]: I didn't verify the existing translation.

File: content/zh/docs/concepts/services-networking/service.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (tanjunchen) [reply content/zh/docs/concepts/services-networking/service.md]: ok.thanks for your good job!

**Pull request merged**